skyportal/models/localization.py

```diff
@@ -141,3 +146,31 @@ def flat(self):
             return healpy.reorder(result, 'NESTED', 'RING')
         else:
             return (self.flat_2d,)
+
+    tiles = relationship("LocalizationTile")
```

Comment:
```suggestion
```

The relationships are not necessary. They are not used in the queries, and I had removed them from my example code.

Removing the relationships will also speed up loading the model instances because it won't load the dependent columns.

Reply:
Great point! Committed.